cli/pkg/pgconn/pgconn.go

```diff
@@ -23,11 +31,18 @@ func OpenConnectionToDB(namespace, name, user, dbname string, remote int) (*pgxp
 	os.Stdout = nil
 	defer func() { os.Stdout = stdout }()
 
-	tspromPods, err := k8s.KubeGetPods(namespace, map[string]string{"app": name + "-promscale"})
+	tspromPods, err := k8s.KubeGetPods(d.Namespace, map[string]string{"app": d.Name + "-promscale"})
 	if err != nil {
 		return nil, err
 	}
 
+	passBytes, err := utils.GetDBPassword(d.SecretKey, d.Name, d.Namespace)
+	if err != nil {
+		fmt.Println(err)
+		return nil, err
```

Comment:
This seems like an anti-pattern. Printing an error (I should say logging it to stdout) and then returning it, where it will most likely be logged later on in the stack.